tools/cmgen/src/CubemapSH.cpp

```diff
@@ -382,7 +381,7 @@ double __UNUSED CubemapSH::Legendre(ssize_t l, ssize_t m, double x) {
 }
 
 // Only used for debugging
-double CubemapSH::TSH(int l, int m, const double3& d) {
+double __UNUSED CubemapSH::TSH(int l, int m, const double3& d) {
```

Comment:
UTILS_UNUSED instead?

Reply:
We have an __UNUSED in cmgen used in many places. I wanted to be consistent.